tests/cri-o/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN yum install -y golang python git gcc automake autoconf libcap-devel \
     chmod 755 /root && \
     (cd /root/go/src/github.com/cri-o && git clone https://github.com/cri-o/cri-o && \
     cd cri-o && \
-    git checkout daab4d1d8431cf3af59855d3ad220e5ded5e5822 && \
+    git checkout tags/v1.18.6 && \
```

Comment:
The following command would checked out only the `v1.18.6 ` branch:
```bash
git clone --depth=1 -b v1.18.6 https://github.com/cri-o/cri-o
```

Reply:
@rst0git Thanks a lot. It seems problem is with the golang version in the CI so i'll bump go to `1.17` and try pointing to the `master` branch. I hope it solves the issue.